bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Deflated.scala

```diff
@@ -0,0 +1,98 @@
+package edu.gemini.spModel.core
+
+import java.io._
+import java.util.Arrays
+import java.util.zip.{ Deflater, Inflater }
+
+/**
+ * A compressed value.
+ * @param data the underlying byte array
+ */
+final class Deflated[A] private (val data: Array[Byte]) extends Serializable {
```

Comment:
Need to make the `data` field private; I can't cope with it being available directly. The only time it's used externally is for serialization so a copy then is ok.